pkgs/applications/audio/mbrola/default.nix

```diff
@@ -65,8 +53,14 @@ let
   };
 
 in
-symlinkJoin {
-  inherit pname version meta;
-  name = "${pname}-${version}";
-  paths = [ bin voices ];
-}
+  runCommandLocal
+    "${pname}-${version}"
+    {
+      inherit pname version meta;
+    }
+    ''
+      mkdir -p "$out/share/mbrola"
+      ln -s '${voices}/data' "$out/share/mbrola/voices"
+      ln -s '${bin}/bin' "$out/"
+    ''
+
```

Comment:
```suggestion
  runCommandLocal "${pname}-${version}" {
    inherit pname version meta;
  } ''
   mkdir -p "$out/share/mbrola"
   ln -s '${voices}/data' "$out/share/mbrola/voices"
   ln -s '${bin}/bin' "$out/"
  ''
```

Reply:
Ah yeah well, I don't care about what code style will be there, so if this is preferred 🤷🏽  (i.e. will wait for opinions of others)

Reply:
We still don't have a uniform code style, so I don't see the point.